### PR TITLE
Save code to world state

### PR DESCRIPTION
**Pull request by norhh**

### CODE REVIEW

File: mythril/laser/ethereum/state/world_state.py [hunk 1]
```diff
@@ -73,26 +73,25 @@ def __copy__(self) -> "WorldState":
...
-                balance=0, address=addr_bitvec.value, dynamic_loader=dynamic_loader
-            )
-        if code is None:
-            code = ""
-        else:
-            code = code.bytecode
-        return code
+            return self.accounts[addr_bitvec.value]
+        if dynamic_loader is None:
+            raise ValueError("dynamic_loader is None")
```
Comment (JoranHonig) [inline mythril/laser/ethereum/state/world_state.py]: If the dynamic loader is none shouldn't the default behaviour be to create a new account (similar to  what would happen if the dynamic loader does not find code for the provided address)?

File: mythril/laser/ethereum/state/world_state.py [hunk 2]
```diff
@@ -73,26 +73,25 @@ def __copy__(self) -> "WorldState":
...
+        return self.create_account(
+            balance=0,
```
Comment (JoranHonig) [inline mythril/laser/ethereum/state/world_state.py]: Should we not take the concrete balance from on-chain?

Comment (norhh) [reply mythril/laser/ethereum/state/world_state.py]: yes, Let's add it after this PR.

File: mythril/laser/ethereum/state/world_state.py (hunk 1 above)
Comment (norhh) [reply mythril/laser/ethereum/state/world_state.py]: A bunch of  opcodes push a symbol to stack if this function raises a ValueError.

File: mythril/laser/ethereum/state/world_state.py (hunk 2 above)
Comment (JoranHonig) [reply mythril/laser/ethereum/state/world_state.py]: 👍

File: mythril/laser/ethereum/state/world_state.py (hunk 1 above)
Comment (JoranHonig) [reply mythril/laser/ethereum/state/world_state.py]: Right, you seem to be correct

**Pull request merged**